src/components/Calculator.jsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
```

Comment:
- [ ] Please remove the `eslint-disable` line and try to fix it manually as this defies the purpose of using linters in the first place.